quaiclient/quaiclient.go

```diff
@@ -135,11 +135,23 @@ type pendingEtxs struct {
 	Etxs []types.Transactions `json:"pendingEtxs"`
 }
 
-func (ec *Client) Append(ctx context.Context, header *types.Header, domPendingHeader *types.Header, domTerminus common.Hash, td *big.Int, domOrigin bool, reorg bool, newInboundEtxs types.Transactions) ([]types.Transactions, error) {
+type entropyStruct struct {
+	Entropy *big.Float `json:"entropy"`
+}
+
+type subSStruct struct {
+	SubS *big.Float `json:"subS"`
+}
+
+type reorgStruct struct {
```

Comment:
Rather than create a bunch of `xyzStruct`s, and parse them independently out of the raw bytes, I'd prefer to see one `appendResponse` struct (or something like that) with all of these members, and decode the response from append in one-shot.